lib/re_captcha/configuration.rb

```diff
@@ -33,7 +36,22 @@ def language_code(locale)
     end
 
     def valid?
-      !private_key.nil? && !public_key.nil?
+      reset_errors
+      set_errors
+      errors.empty?
+    end
```

Comment:
I would agree with you in an other scenario but here we are already pretty coupled to rails https://github.com/textmaster/re_captcha/blob/master/lib/re_captcha/engine.rb#L4

Beside, requiring activemodel will not require activesupport code and even if it does, both of them are already loaded by rails anyway.